### DISCUSSION TITLE
source map for large first-party JavaScript not working for the config?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#64309] [Help] [answered]
### Summary

Google lighthouse error:
![image](https://github.com/vercel/next.js/assets/93617433/b38f664f-85f2-4bbd-91a1-007f7bc1d74a)

The code I have:

```
const nextConfig = {
  reactStrictMode: true,
  productionBrowserSourceMaps: true,
/// and a bunch of redirects
}

module.exports = nextConfig;

```

The link of the file:
https://github.com/Elindo586/nextprojectb/blob/main/next.config.js

I ran npm run build (working good), then npm run dev and went to check the lighthouse and still gave me an error for the source map.

**Comment by icyJoseph** [comment, marked as answer]
The source maps are for prod builds, do `npm run build`, and then `npm run start`, you can run lighthouse on that instead.

**Reply by Elindo586** [reply]
Well, that error went away...

And my performance from lighthouse went from the 40-50's to 99 !!

My goodness, seems like I am almost done with 90% of the Lighthouse errors.

**Reply by icyJoseph** [reply]
Fantastic! Do remember to also track web-vitals. There's a Chrome extension that can measure that. It won't be exactly what users see, but very close.

- https://web.dev/articles/debug-cwvs-with-web-vitals-extension
